pkg/storaged/block/format-dialog.jsx

```diff
@@ -416,6 +424,18 @@
             disable_on_error: usage.Teardown,
             action: function (vals) {
                 const mount_now = vals.variant != "nomount";
+                let type = vals.type;
+                let partition_type = null;
+
+                if (type == "efi") {
+                    type = "vfat";
+                    partition_type = block_ptable.Type == "dos" ? "0xEF" : "c12a7328-f81f-11d2-ba4b-00a0c93ec93b";
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19826-20240109-133522-cb1438ac-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)